### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by wperron**
Add link pointing to the Proposals in the Discussions tabl

### CODE REVIEW

**Comment by kt3k** [comment]
With this change, the UI changes like this.

スクリーンショット 2021-06-15 11 09 06

It feels like we're only accepting feature request. What do you think? Probably it might better to have 'bug report' template as well?

ref: https://github.com/kt3k/test-issue-tmpl

**Comment by wperron** [comment]
> With this change, the UI changes like this.
> 
> スクリーンショット 2021-06-15 11 09 06
> 
> It feels like we're only accepting feature request (thought there is a small link below saying 'open a blank issue', people might miss it). What do you think? Maybe it might better to have 'bug report' template as well?
> 
> ref: https://github.com/kt3k/test-issue-tmpl

Oh wow, yup, I'll add one this PR as well

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+<!-- NOTE: for larger scale feature requests like additions to the standard library, consider opening a new discussion under the "Proposals" category over here: https://github.com/denoland/deno_std/discussions/new?category=proposals -->
+
+**Is your feature request related to a problem? Please describe.** A clear and
+concise description of what the problem is.
+
+**Describe the solution you'd like** A clear and concise description of what you
+want to happen.
+
```
Comment (getspooky) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]: **Teachability, Documentation, Adoption, Migration Strategy**

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+**Describe alternatives you've considered** A clear and concise description of
+any alternative solutions or features you've considered.
+
```
Comment (getspooky) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]: **Teachability, Documentation, Adoption, Migration Strategy**

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+**To Reproduce** Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior** A clear and concise description of what you expected to
+happen.
+
```
Comment (getspooky) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: **Possible Solution** Suggest possible solutions

**Review by getspooky — CHANGES_REQUESTED** [review]

**Comment by getspooky** [comment]
@wperron It would be a good idea to add template for `Regression` ?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (getspooky) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]:
```suggestion

**Possible solution** Suggest solution. 

```

Comment (wperron) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: If there's a possible solution, it fits in the "additional info" section, or even a PR

**Comment by wperron** [comment]
> @wperron It would be a good idea to add template for `Regression` ?

Let's start with Bug Report and Feature Request and see how it goes first

**Comment by kt3k** [comment]
> It would be a good idea to add template for Regression ?

I feel `Regression` is included in `Bug` and it will be confusing if we have both.

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**